MaxText/configs/base.yml

```diff
@@ -40,6 +40,7 @@ gcs_metrics: False
 # Activation dtypes.
 dtype: "bfloat16"
 int8_training: False
+use_dqdg: False
```

Comment:
Add a comment that it has effect only if int8_training=True on this and 2 flags belof